core/domain/config_domain.py

```diff
@@ -463,3 +464,28 @@ def get_all_config_property_names(cls):
         'Enable sending Contributor Dashboard reviewers email notifications '
         'about suggestions that need review. The default value is false.'
     ), False)
+
+NOTIFY_ADMINS_SUGGESTIONS_WAITING_TOO_LONG_IS_ENABLED = ConfigProperty(
+    'notify_admins_suggestions_waiting_too_long_is_enabled', BOOL_SCHEMA,
+    (
+        'Enable sending admins email notifications if there are Contributor '
+        'Dashboard suggestions that have been waiting for a review for more '
+        'than %s days. The default value is false.' % (
+            suggestion_models.SUGGESTION_REVIEW_WAIT_TIME_THRESHOLD_IN_DAYS)
+    ), False)
+
+NOTIFY_ADMINS_REVIEWERS_NEEDED_IS_ENABLED = ConfigProperty(
+    'notify_admins_reviewers_needed_is_enabled', BOOL_SCHEMA,
+    (
+        'Enable sending admins email notifications if Contributor Dashboard '
+        'reviewers are needed in specific suggestion types. The default value '
+        'is false.'
+    ), False)
+
+MAX_NUMBER_OF_SUGGESTIONS_PER_REVIEWER = ConfigProperty(
+    'max_number_of_suggestions_per_reviewer',
+    FLOAT_SCHEMA,
```

Comment:
Why float? I was expecting this to be Int*

Reply:
Changed!